includes/functions.php

```diff
@@ -2493,3 +2493,27 @@ function ap_get_current_timestamp() {
 
 	return $local_time->getTimestamp() + $local_time->getOffset();
 }
+
+/**
+ * Merges user defined array arguments into the defaults array,
+ * which works with arrays only.
+ *
+ * @param array $args     Value to be merged with the defaults array.
+ * @param array $defaults Array that serves as the defaults.
+ *
+ * @return array Merged user defined values with the default array.
+ * @since 4.4.0
+ */
+function ap_parse_args( $args, $defaults ) {
```

Comment:
@mebishalnapit Add type casting for the two arguments.

Reply:
@rahularyan Added.